shared/actions/wallets.js

```diff
@@ -645,7 +645,7 @@ const navigateToAccount = (state, action) => {
   }
   const wallet = isMobile
     ? [Tabs.settingsTab, SettingsConstants.walletsTab]
-    : [{props: {}, selected: Tabs.walletsTab}, {props: {}, selected: null}]
+    : [{props: {}, selected: Tabs.walletsTab}]
```

Comment:
This extra path component doesn't seem to do anything on the old nav, but goes down a path in oldActionsToNewActions that ends up returning nothing.